opencanary/modules/samba.py

```diff
@@ -15,12 +15,16 @@ def __init__(self, logFile=None, logger=None):
 
         def handleLines(self, lines=None):
             #samba4 re
-            audit_re = re.compile(r'.*smbd_audit: (.*)')
+            audit_re = re.compile(r'.*smbd_audit: ((?:[^|]*\|){12}[^|]*)$')
```

Comment:
Im nervous of this regex.. We would rather get more matches that come from `smbd_audit`. The issue im thinking of is different smb versions and implementations such that your more restrictive regex may not match. 

I think your line below; checking `matches is None` should solve the issue you experienced in #67

Reply:
This would resolve this issue, however I also experienced several instances where the usual regex _did_ match but with more fields (ex: open, close), which makes this module break as well. For this reason I also changed the regex to make sure it doesn't match lines not conforming to the 13-field data format expected by the lines below, but this could also be enforced by using try/catch constructions like some of the other modules do.
However, since that strictly isn't the same issue I can open a seperate issue/pull for this if desired.